austation/code/modules/coilgun/_coilgun.dm

```diff
@@ -0,0 +1,96 @@
+
+// TheFakeElon's funni high-velocity-projectile launcher.
+
+/obj/structure/disposalpipe/coilgun
+	name = "coilgun tube"
+	desc = "An electromagnetic tube that allows the safe transportation of high speed magnetic projectiles"
+	icon = 'austation/icons/obj/atmospherics/pipes/disposal.dmi'
+	layer = 2.45
+	initialize_dirs = DISP_DIR_FLIP
+	coilgun = TRUE
+
+/obj/structure/disposalpipe/coilgun/magnetizer
+	name = "magnetizer"
+	desc = "A machine that glazes inserted objects with neodymium, making the object magnetive"
+	icon_state = "magnet"
+
+/obj/structure/disposalpipe/coilgun/magnetizer/transfer(obj/structure/disposalholder/H)
+	if(H.contents.len) // is there an object in here?
+		update_icon()
+		var/obj/effect/hvp/boolet
+		for(var/atom/movable/AM in H.contents)
+			if(AM == boolet)
+				continue
+			if(H.contents.len > 1)
+				visible_message("<span class='warning'>\The [src] can't magnetize more than one object at a time!</span>")
+				AM.forceMove(get_turf(src))
+				continue
+			else
+				boolet = new(H)
+				boolet.name = AM.name
+				boolet.desc = AM.desc
+				boolet.icon = AM.icon
+				boolet.icon_state = AM.icon_state
+				boolet.p_speed = 1
+				AM.loc = boolet //put the original inserted objected inside the coilgun projectile
+				boolet.apply_special(AM, TRUE)
+			if(isliving(AM))
+				var/mob/living/L = AM
+				L.adjustBruteLoss(10)
+				if(ishuman(L) && !isdead(L))
+					L.Paralyze(amount = 50, ignore_canstun = TRUE)
+					L.emote("scream")
+					boolet.mass = 5
+					sleep(30)
+				else
+					boolet.mass = 3
```

Comment:
humans should give the same mass, alive or dead

Reply:
that's no fun